treemacs-customization.el

```diff
@@ -70,6 +70,45 @@ times."
   :type 'boolean
   :group 'treemacs-configuration)
 
+(defun treemacs-windows (&optional frame)
+  "Return a list of Treemacs windows across all frames or just from optional FRAME."
+  (delq nil (mapcar (lambda (win)
+		      (when (eq (buffer-local-value 'major-mode (window-buffer win))
+				'treemacs-mode)
+			win))
+		    (window-list-1 nil nil (or frame t)))))
+
+(defcustom treemacs-window-location 'left
```

Comment:
That's already available under `treemacs-position`. Without the extra set option, but I don't imagine it's something that'll change often.